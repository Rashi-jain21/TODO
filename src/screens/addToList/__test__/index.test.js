import {Provider} from 'react-redux';
import configureStore from 'redux-mock-store';
import renderer from 'react-test-renderer';
import AddToList from '..';

const mockStore = configureStore();

it('renders with snapshot with no data', () => {
  const initialState = {todos: []};
  const tree = renderer
    .create(
      <Provider store={mockStore(initialState)}>
        <AddToList />
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
