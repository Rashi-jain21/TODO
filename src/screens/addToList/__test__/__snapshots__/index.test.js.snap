// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with snapshot with no data 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 0.5,
      "justifyContent": "space-between",
      "padding": 20,
    }
  }
>
  <View
    style={
      Object {
        "borderColor": "#4290f5",
        "borderRadius": 10,
        "borderWidth": 1,
        "height": 40,
        "justifyContent": "center",
        "margin": 10,
        "padding": 10,
        "width": "90%",
      }
    }
  >
    <TextInput
      keyboardType="default"
      maxLength={50}
      onChangeText={[Function]}
      placeholder="enter text"
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
        }
      }
      value=""
    />
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#4290f5",
        "borderRadius": 10,
        "height": 40,
        "justifyContent": "center",
        "width": 100,
      }
    }
  >
    <View
      accessibilityRole="button"
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Array [
            Object {},
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#007AFF",
                "fontSize": 18,
                "margin": 8,
                "textAlign": "center",
              },
              Object {
                "color": "white",
              },
            ]
          }
        >
          Done
        </Text>
      </View>
    </View>
  </View>
</View>
`;
